src/inject/inject.js

```diff
@@ -12,32 +12,32 @@ chrome.extension.sendMessage({}, function(response) {
 			inject('main');
 			injectIf('viewLatestComments');
 			injectIf('showSpoilers');
-			injectIfUrl('banRequestsForm', '/user/');
```

Comment:
not from those you poopoo
this matches /user/Whatever
but it will still work fine with /userWhatever so ill merge